packages/parcel-transformer/src/transformer.tsx

```diff
@@ -111,14 +115,19 @@ export default new Transformer<UserlandOpts>({
     return [asset];
   },
 
-  generate({ ast, asset }) {
+  async generate({ asset, ast }) {
```

Comment:
Is there a reason for this method to be `async`? I don't see any async operations being call or is some we wrap the return in a promise?

Reply:
yeah, the TypeScript types were updated so we need to return a promise now